webui/react/src/services/api.ts

```diff
@@ -1,27 +1,16 @@
 import * as Api from 'services/api-ts-sdk';
 import * as Config from 'services/apiConfig';
-import {
-  CommandIdParams, CreateExperimentParams, EmptyParams, ExperimentDetailsParams, ExperimentIdParams,
-  GetCommandsParams, GetExperimentParams, GetExperimentsParams, GetJupyterLabsParams,
-  GetResourceAllocationAggregatedParams, GetShellsParams, GetTemplatesParams, GetTensorBoardsParams,
-  GetTrialsParams, LaunchJupyterLabParams, LaunchTensorBoardParams, LoginResponse,
-  PatchExperimentParams, SingleEntityParams, TaskLogsParams, TrialDetailsParams,
-} from 'services/types';
+import * as ServiceType from 'services/types';
```

Comment:
nit: since it'd be something repetitive that we'll see very frequently I'd go with a shorter name eg SType? It should help with line length limits

Reply:
or maybe ServiceT and T (for Type). The upper CamelCase nature of type names kind of give you a hint of what they are.

Reply:
Good call. I shortened `ServiceType` to just `Service` to keep it in the same spirit as `Api` and `Type`.